### ISSUE TITLE
Error in sending email from ERPNext

### ISSUE THREAD

**Issue by unidow** [frappe/erpnext#20699]
Hi,

while sending email from ERPNext, i am getting the following error message:

**Encryption key is invalid, Please check site_config.json**

let me know how to fix this.

**Comment by hrwX** [comment]
Did you restore the backup to another site ??

**Comment by unidow** [comment]
No. We did not do anything which might have caused this error. Is it related to an upgrade issue?
